### DISCUSSION TITLE
What's  the use of this?

### DISCUSSION THREAD

**Discussion by beedell-roke** [openSUSE/opi#122] [Q&A] [answered]
It just seems like an alternative stub for `zypper`. I read the README, but that didn't appear to answer this.

http://reddit.com/comments/11oka0z/comment/jbu53e5/?utm_source=share&utm_medium=web2x&context=3 suggested this. I'd not heard of it previously. To get `firefox` to work with MP4 video, I used a .ymp file from an official-looking OpenSUSE website that some documentation suggested.

**Comment by aylen384** [comment, marked as answer]
Hi,

in the end opi just calls zypper commands, yes.
But it makes it easier to install packages from repos that are not yet added to the system.
It will then add the repos of packages from the opensuse build service, the community drive packman repo with packages that suse cannot ship for eg. for patent reasons or eg. the proprietary google repos for chrome.
In the case of `opi codecs` it installs packages from the packman repo and the opensuse openh264 codecs repo.
This is a bit tricky, so running `opi codecs` is easier for a user than adding the respective repos.
Here you can see, what it will do: https://github.com/openSUSE/opi/blob/master/opi/plugins/packman.py
